client/src/bundles/FieldEditor.js

```diff
@@ -26,11 +26,13 @@ jQuery.entwine('ss', ($) => {
             return;
           }
           case 'SilverStripe\\UserForms\\Model\\EditableFormField\\EditableFieldGroup': {
-            thisLevel = ++depth;
```

Comment:
This doesn't look right, unless this was not supposed to increment before assigning.

Reply:
Good spotting!

Just pushed a fixed.